pages/sigmaker.tsx

```diff
@@ -42,37 +51,119 @@ const Sigmaker: NextPage = () => {
     />
   );
 
+  const createSignature = () => {
+    if (signatureData) {
+      return (
+        <div>
+          <Typography variant="h4">
+            Paste this into Gmail!
+          </Typography>
+          <Container>
+            <table cellPadding={0} cellSpacing={0} className="table">
+              <tbody>
+                <tr>
+                  <td valign="top" className="logoContainer">
+                    <img className="logo" id="preview-image-url" src="https://tools.hackbeanpot.com/assets/logos/2018_logo_400px.png" />
```

Comment:
Since this is referencing an image in the original tools repo, once we switch the tools.hackbeanpot to point to this repository, the src will not load. Instead, let's download this image or copy it from the previous repo and add it the source code for this project! You can add it to /public/assets and the icons.js file. Then import {icons} and use the src in the image tag.